Binary files apache2-2.4.10.orig/.git/index and apache2-2.4.10/.git/index differ
diff -urBN apache2-2.4.10.orig/debian/config-dir/mods-available/authnz_fcgi.load apache2-2.4.10/debian/config-dir/mods-available/authnz_fcgi.load
--- apache2-2.4.10.orig/debian/config-dir/mods-available/authnz_fcgi.load	1970-01-01 01:00:00.000000000 +0100
+++ apache2-2.4.10/debian/config-dir/mods-available/authnz_fcgi.load	2014-09-26 09:02:39.190581344 +0200
@@ -0,0 +1,2 @@
+# Depends: proxy_fcgi
+LoadModule authnz_fcgi_module /usr/lib/apache2/modules/mod_authnz_fcgi.so
diff -urBN apache2-2.4.10.orig/debian/rules apache2-2.4.10/debian/rules
--- apache2-2.4.10.orig/debian/rules	2014-09-26 09:22:31.226230405 +0200
+++ apache2-2.4.10/debian/rules	2014-09-26 09:00:21.934621283 +0200
@@ -86,7 +86,7 @@
 		--with-pcre=yes \
 		--enable-pie \
 		--enable-mpms-shared=all \
-		--enable-mods-shared="all cgi" \
+		--enable-mods-shared="all cgi authnz-fcgi" \
 		--enable-mods-static="unixd logio watchdog version" \
 		CFLAGS="$(AP2_CFLAGS)" CPPFLAGS="$(AP2_CPPFLAGS)" LDFLAGS="$(AP2_LDFLAGS)" \
 		LTFLAGS="$(AP2_LTFLAGS)"
